jobs/release.Jenkinsfile

```
@@ -96,11 +99,16 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             """)
         }
 
-        def basearches = ""
-        if (params.ARCHES == '') {
-            basearches = shwrapCapture("jq -r '.builds | map(select(.id == \"${params.VERSION}\"))[].arches[]' builds/builds.json").split()
-        } else {
-            basearches = params.ARCHES.split()
+        def basearches = params.ARCHES.split()
+        def builtarches = shwrapCapture("jq -r '.builds | map(select(.id == \"${params.VERSION}\"))[].arches[]' builds/builds.json").split()
+        if builtarches.toSet() != basearches.toSet() {
```

Hmm, shouldn't this be a subset check though?

Ahh, looks like `containsAll()` exists on any Iterable: https://docs.groovy-lang.org/latest/html/groovy-jdk/java/lang/Iterable.html#containsAll(java.lang.Object[]).

So I think this could be

```suggestion
        if !builtarches.containsAll(basearches) {
```

(Double-check in the script console first.)